src/Dumplie/Application/Query/Customer/CartQuery.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Query\Customer;
+
+use Dumplie\Application\Query\Customer\Result\Cart;
+
+interface CartQuery
+{
+    /**
+     * @param string $cartId
+     *
+     * @return Cart
+     */
+    public function getById(string $cartId) : Cart;
```

Comment:
As you can see there is no single getter for any property at any object, Queries are different case